io.podman_desktop.PodmanDesktop.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=Podman Desktop
```

Comment:
a linux desktop application doesn't come with a .desktop so that you hae to ship it?

Reply:
ok so I can drop this file ?

Reply:
you tell me. but if there is one upstream definitely.

Reply:
so, no there is no one defined upstream this is why it's there

Reply:
but they have an appdata. Logical.
</sarcasm>

Reply:
note that since they seem to have a flaptak provided upstrean, I doubt there is no .desktop, because it wouldn't work.

Reply:
if you really have to create it you should make sure it's submitted upstream.

Reply:
@hfiguiere the .desktop file is generated automatically by electron-builder

Reply:
then you don't need one here. As I originally said.

Reply:
it's generated when electron builder is generating a single bundle flatpak but here we are not using `flatpak` output as we're using flatpak-builder so there is none upstream and we need one here

Reply:
also we want to remove .flatpak file upstream and only use flathub

Reply:
then maybe all of this should be fixed first.

Reply:
still no word on using the upstream generated .desktop file?

Reply:
we can't use it as it's generated when using `flatpak` target of electron-builder and we can't use that target here as we're using a dedicated yaml file with instructions on how to build the bundle

Reply:
but in upcoming releases, we may store it upstream, etc.

Reply:
why not generating it on on flatpak target? A desktop app without a .desktop file is literally a bug.

Reply:
electron-builder is generating it during the generation of the flatpak.
Before we don't have the desktop file
and after, all temp folders are cleaned, we only have the `.flatpak` file

so I can't say: generate the desktop file only or something like that.

and as I said, I could store it upstream but it needs to wait a new version of the tool.

Egg and chicken issue